snuba/datasets/transactions_processor.py

```diff
@@ -133,7 +134,12 @@ def process_message(self, message, metadata) -> Optional[ProcessedMessage]:
                 (
                     processed["measurements.key"],
                     processed["measurements.value"],
-                ) = extract_nested(measurements, lambda value: float(value["value"]))
+                ) = extract_nested(
+                    measurements,
+                    lambda value: float(value["value"])
+                    if isinstance(value["value"], numbers.Number)
+                    else None,
```

Comment:
This unfortunately will not work. The column is not nullable on Clickhouse, adding a NULL will make the write fail.
Any reason we cannot we make measurements consistent with other nested columns on relay (which means a NULL measurements gets removed) ? I would say we should try to make them consistent.

Reply:
@fpacifici That's what I thought as well; but it should be fine because of this check on the `extract_nested ()` function: https://github.com/getsentry/snuba/blob/95f0ac7253cb8f9e79c17890d4a522ee4641ed26/snuba/datasets/events_format.py#L55-L56

Reply:
mh ok then